src/libs/ck-libs/NDMeshStreamer/NDMeshStreamer.ci

```diff
@@ -13,6 +13,7 @@ module NDMeshStreamer {
 
   template<class dtype, class RouterType>
   group [migratable] MeshStreamer {
+    entry void localDeliver(size_t size, const char data[size], CkArrayIndex arrayId, int sourcePe);
```

Comment:
Making `localDeliver` an entry method causes `size` and `data` parameters to switch places (since `size` needs to come before `data`), which can be seen in multiple places of this PR.